bot/requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 pre-commit==1.18.2
 pytest==5.1.1
 responses==0.10.6
+pytest-responses==0.4.0
```

Comment:
We try to keep this file ordered alphabetically: please move that line before `responses`

Reply:
fixed